### PR TITLE
Update LIT6719prologue.xml

### PR DESCRIPTION
**Pull request by karljonaskarlsson**
I created the work record (with Zhenia's help for the first steps).

### CODE REVIEW

**Review by thea-m — APPROVED** [review]
Everything seems fine to me!

**Comment by thea-m** [comment]
https://github.com/BetaMasaheft/Documentation/issues/2236
It is always better to link to the work creation issue for documentation

**Comment by SophiaD-M** [comment]
May I suggest to start with capital P - LIT6719Prologue ?
I somehow think that this is usually the case? Or am I wrong?

**Comment by thea-m** [comment]
Yes, it's the convention, but there is no problem if it's not respected in one case. Better to leave it as created and not change an existing ID

**Comment by SophiaD-M** [comment]
but was it not created just now? Would this not be the best possible time to change the ID?

File: new/LIT6719prologue.xml [hunk 1]
```diff
@@ -30,7 +32,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p/>
+                <p>A prologue to the Book of Psalms ascribed to Eusebius of Caesarea. It usually consists of an introduction,
+                    a list of the authors of psalms (frequently with the number of psalms ascribed to each of them
+                    appearing next to the name), and an section regarding the naming of the Book of Psalms, followed
```
Comment (MassimoVilla) [inline new/LIT6719prologue.xml:37]: a section

**Pull request merged**